api/python/t4/packages.py

```diff
@@ -647,9 +647,14 @@ def _fix_sha256(self):
         if not entries:
             return
 
-        physical_keys = (entry.physical_keys[0] for entry in entries)
-        total_size = sum(entry.size for entry in entries)
-        results = calculate_sha256(physical_keys, total_size)
+        physical_keys = []
+        sizes = []
+        for entry in entries:
+            physical_keys.append(entry.physical_keys[0])
+            sizes.append(entry.size)
+
+        total_size = sum(sizes)
```

Comment:
Don't need this anymore

Reply:
But you need `total_size` for `tqdm(desc="Hashing", total=total_size, unit='B', unit_scale=True)`?